INSTALL.md

```diff
@@ -491,15 +491,15 @@ finite) which are required for the available algorithms. See section
 "ApplicableMethod" and "KnownPropertiesOfObject" of the GAP Reference
 manual.
 
-Problems specific to Windows
-
 ### The ^-key or "-key cannot be entered.
 
 This is a problem if you are running a keyboard driver for some non-english
 languages. These drivers catch the ^ character to produce the French
-circumflex accent and do not pass it properly to GAP. No fix is known. (One
+circumflex accent and do not pass it properly to GAP. For OS X users, as a workaround please refer to the section "GAP for OS X" below for information on how to install readline and section 5 on how to recompile GAP, for windows no fix is known. (One
 can type POW(a,b) for a^b.)
 
+Problems specific to Windows
```

Comment:
This is not your fault, but this line is really weird. Was this perhaps meant to be a section headline? If so, perhaps it should be `## Problems ...` ? Perhaps @alex-konovalov knows?

Reply:
I wondered about this as well.  I can change it if that's okay.
```suggestion
##Problems specific to Windows
```

Reply:
There should be a space after `##`